pkg/controller/account/account_controller.go

```diff
@@ -91,6 +91,17 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 		scheme:           mgr.GetScheme(),
 		awsClientBuilder: &awsclient.Builder{},
 	}
+
+	configMap, err := controllerutils.GetOperatorConfigMap(reconciler.Client)
+	if err != nil {
+		log.Error(err, "failed getting shard name from configmap data")
```

Comment:
here it's a failure in retrieving the configmap and the shard name is the next section